coffeecard/CoffeeCard.Models/DataTransferObjects/v2/User/UserLoginResponse.cs

```diff
@@ -0,0 +1,24 @@
+namespace CoffeeCard.Models.DataTransferObjects.v2.User
+{
+    /// <summary>
+    /// User login response object
+    /// </summary>
+    /// <example>
+    /// {
+    ///     "jwt": "hidden",
+    ///     "refreshToken": "hidden"
```

Comment:
I think we usually use `"[no example given]"` instead of "hidden"

Reply:
Done in https://github.com/AnalogIO/analog-core/pull/289/commits/c175b82c8e42d5a0276211d1b8b585fc0d338862